x/auth/client/cli/tx_sign.go

```diff
@@ -188,8 +185,8 @@ be generated via the 'multisign' command.
 	}
 
 	cmd.Flags().String(flagMultisig, "", "Address of the multisig account on behalf of which the transaction shall be signed")
-	cmd.Flags().Bool(flagAppend, true, "Append the signature to the existing ones. If disabled, old signatures would be overwritten. Ignored if --multisig is on")
-	cmd.Flags().Bool(flagSigOnly, false, "Print only the generated signature, then exit")
+	cmd.Flags().Bool(flagOverwrite, false, "Append the signature to the existing ones. If disabled, old signatures would be overwritten. Ignored if --multisig is on")
```

Comment:
```suggestion
	cmd.Flags().Bool(flagOverwrite, false, "Append the signature to the existing ones. If enabled, existing signatures would be overwritten. Ignored if --multisig is on")
```

On `Ignored if --multisig is on`, are we sure that that is truthful @robert-zaremba (can't remember by heart now)?

Reply:
I will change the wording: 
_Overwrite existing signatures with a new one. If disabled, new signature will be appended_

Re `--multisig` - we can enable it because now we are using `SignatureV2`.